### DISCUSSION TITLE
First impressions

### DISCUSSION THREAD

**Discussion by thorbeck** [community/community#7634] [Copilot Conversations]
Just got it installed and continued on mocking up a component.
How it understands what I am doing is impressive.
I'm using typescript, and faker.js for random data.

Example of suggestion:
![image](https://user-images.githubusercontent.com/798384/141468610-f32ec8b5-a239-40de-9168-805f9fcfc469.png)

Just started using it, but I can clearly see how helpful this can be with the more mundane tasks, giving me more time to focus on the important bits.

Thanks for letting me in!

**Comment by Urethramancer** [comment]
I've found it to even write some important bits for me. I've been writing a bit of hairy SQL this weekend, which it handled pretty well. Once I corrected minor details where it didn't get it 100% correct, it continued along those lines for related functions.
